core/primitives/src/bandwidth_scheduler.rs

```diff
@@ -35,16 +35,6 @@ impl BandwidthRequests {
     pub fn empty() -> BandwidthRequests {
```

Comment:
Instead of using `empty`, derive `Default` for BandwidthRequests and use that instead.